src/delete.js

```diff
@@ -0,0 +1,10 @@
+const removeCompleted = document.querySelector('.clear-btn');
+export default function complete(myToDolist, render) {
+  removeCompleted.addEventListener('click', () => {
+    const temp = myToDolist.ToDo.filter(({ doneBox }) => !doneBox);
+    myToDolist.SetToDo(temp);
+    myToDolist.reorder();
+    myToDolist.SaveToDolistLocal();
+    render();
+  });
+}
```

Comment:
- Kindly make sure in these files you add methods related to the status updates (completed: true/false) as was required in the project so that you will be able to select the completed tasks and unselected them :+1: 
-  Kindly make sure It has an event listener on the checkbox (`change`). as it was required in the project  to view of  the project selected  :grinning: :grinning: :grinning: 
- The task object's values for `completed` is updated upon user actions.  :grinning: :grinning: 
- Kindly for  a function for the "Clear all completed" button using the `Array. filter()` method is implemented. :grinning: :grinning: :grinning: 
- Kindly  make sure the updated array of items is stored in local storage, so the user gets the correct list values after the page reloads. Make sure to reload the page to test the array is stored. :grinning: :grinning: :grinning:

### Check the expected results below  

![image](https://user-images.githubusercontent.com/95297251/193414756-b3dbe116-863d-4d2d-8d0b-f83106df422a.png)